### ISSUE TITLE
fake news

### ISSUE THREAD

**Issue by Mallmayank** [Mallmayank/fake-news-detection#1]
import pandas as pd
from sklearn.feature_extraction.text import TfidfVectorizer
from sklearn.linear_model import LogisticRegression

fake = pd.read_csv("Fake.csv")
real = pd.read_csv("True.csv")

fake["label"] = 0
real["label"] = 1

data = pd.concat([fake, real])
data = data[["text","label"]]

vectorizer = TfidfVectorizer(max_features=5000)

X = vectorizer.fit_transform(data["text"])
y = data["label"]

model = LogisticRegression()
model.fit(X, y)

print("Model trained successfully")
